src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <AssemblyName>func</AssemblyName>
     <RuntimeIdentifiers>win-x64;win-x86;linux-x64;osx-x64</RuntimeIdentifiers>
-    <Version>2.0.1.34-beta</Version>
+    <Version>2.0.1.35-beta</Version>
```

Comment:
why are you updating the version? 34 is the correct next version. https://github.com/Azure/azure-functions-core-tools/releases